### PR TITLE
refactor: Rename firebase client type

### PR DESCRIPTION
**Pull request by erezrokah**
Follow up on https://github.com/cloudquery/cloudquery/pull/618

### CODE REVIEW

**Review by disq — APPROVED** [review]

**Review by roneli — CHANGES_REQUESTED** [review]
Move this under registry, moreover merge this after #623  is merged.

**Comment by erezrokah** [comment]
@roneli rebased the PR

**Review by roneli — APPROVED** [review]
I think this should maybe move to hub package as its only related to it, maybe in a separate PR.

**Pull request merged**
